HHS/Head-Start-TTADP
# Remove html from csv

## Description of change

Convert HTML to plain text in CSV download

## How to test

Download CSVs with complex `context` and `ttaProvided` fields. Verify no HTML is being sent back in the CSV.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-477

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
